saidump/Makefile.am

```diff
@@ -2,8 +2,11 @@ AM_CXXFLAGS = $(SAIINC) -I$(top_srcdir)/lib
 
 bin_PROGRAMS = saidump
 
-saidump_SOURCES = saidump.cpp
+saidump_SOURCES = main.cpp saidump.cpp
```

Comment:
```
[saidump/Makefile.am](https://github.com/sonic-net/sonic-sairedis/pull/1391/files#diff-c55234eacbecf2fcc92afbe78f4f89e0b3845782ede0dd9b7a8a9b9d9556205b)
 100644 → 100755
```

why attributes on Makefile are changed to 755 ?

Reply:
Fixed.